ios/Classes/DocumentViewFactory.m

```diff
@@ -0,0 +1,28 @@
+#import "FlutterDocumentView.h"
+#import "PdftronFlutterPlugin.h"
+#import "PTPluginUtils.h"
+
+@implementation DocumentViewFactory {
```

Comment:
This seems like an odd class to me. Why does it exist? One method calls a static method on another class. The other method calls a static method too, but uses this class' one member (instance) variable. I guess the callers don't have access to the `FlutterBinaryMessenger` when they call the create method?

Reply:
I deleted that class and merged it into PTFlutterPlugin